### DISCUSSION TITLE
Updated to 0.17 and 2m band says TX Disable

### DISCUSSION THREAD

**Discussion by jezasaurus** [egzumer/uv-k5-firmware-custom#146] [General]
pretty pointless to run this new firmware and have the 2m band TX locked?

is there an option to open this ?

**Comment by egzumer** [comment]
It is not disabled. You messed up your configuration. One point of failure is AM/USB turned on, the other is hidden F Lock menu option.

**Reply by jezasaurus** [reply]
All I did was open a brand new radio, flash from your most recent update I didn't choose anything so how can I have messed it up?

**Reply by OE1MWW** [reply]
just to upload the new modified firmware is not enough. load a frequency list etc. and all will be fine.

**Comment by ErikS-web** [comment]
My guess, AM is on....

Check Main-menu number 13.

See Manual in the Wiki at
[13. `Demodu`](https://github.com/egzumer/uv-k5-firmware-custom/wiki/Menu#main-menu) - demodulator mode, default is FM, AM/USB can be used for listening only

**Comment by f5gkw** [comment]
In (IU0) UJV firmware the 2 m tx, even in dsb, is available.
73
F5GKW

**Reply by OE1MWW** [reply]
interresting. You ran into problems neither I nor others having encountered. 
And there are several users of 0.17 / 0.18 in my club.

**Comment by ErikS-web** [comment]
I think we all had that on the 1st use. 
But RTFM (= read the manual) is very useful..... What do all those symbols on the display mean and how can I adjust it....

But for now ... topic starter seems to have been reading for 18 hours ;-)

**Reply by jezasaurus** [reply]
I don't expect to have to flash new firmware and have my radio not TX on 2m 🤷🏻‍♂️

Not rocket science is it

**Reply by ErikS-web** [reply]
Than show us more info.

**Reply by jezasaurus** [reply]
What exactly do you need to see? I took a fresh radio out the box, flashed it to 0.17 as that was the newest one available and it wouldn't tx on 2m, said TX disable, the remedy is to open the menu EVERYTIME you turn it on to disable TX

No thanks, I've gone with another firmware now
